app/(tabs)/personalInformation.tsx

```diff
@@ -0,0 +1,177 @@
+import { Box, HStack, ScrollView, Text } from "@gluestack-ui/themed";
+import { SecondaryPageHeader } from "@/components/secondaryPageHeader";
+import TextInput from "@/components/uikit/Input/TextInput";
+import DateInput from "@/components/uikit/Input/DateInput";
+import RadioButton from "@/components/uikit/Input/RadioButton";
+import { SolidLong } from "@/components/uikit/Buttons/SolidLong";
+import { User } from "@/components/icons/user";
+import { Call } from "@/components/icons/call";
+import { Mail } from "@/components/icons/mail";
+import { Location } from "@/components/icons/location";
+import { personalInformationValidationSchema } from "@/utils/validationSchemas";
+import { Formik } from "formik";
+import { useState } from "react";
+
+const defaultValues = {
+  completeName: "",
+  personalNumberIdentification: "",
+  phoneNumber: "",
+  email: "",
+  address: "",
+  gender: "",
+};
+
+export default function PersonalInformation() {
+  const [isLoading, setIsLoading] = useState(false);
+  const onSubmit = () => {
+    setIsLoading(true);
```

Comment:
`setIsLoading(false)`  is not missing?

Reply:
Oh, yes, but first, shouldn't it provide that list of data to firestore ? This is just there for now...